ansible/config_sonic_basedon_testbed.yml

```diff
@@ -79,8 +79,14 @@
     delegate_to: localhost
     when: "VM_topo | bool"
 
-  - name: find interface name mapping and individual interface speed if defined
+  - name: find interface name mapping and individual interface speed if defined from dut
     port_alias: hwsku="{{ hwsku }}"
+    when: deploy is defined and deploy|bool == true
```

Comment:
Hi @lguohan 
It is a bit confusing to me that this is done only when the `deploy` is true.
IMO, `deploy` means whether the newly generated minigraph.xml is going to be deployed on the DUT. But here it is going to fetch the `port_config.ini` file from the DUT. I think they are not relevant. 
Sometimes we just want to generate a minigraph file based on the `port_config.ini` on the DUT and will deploy the minigraph.xml in another step. In that case, this can fail the process.
Is it possible to introduce another variable to control this behavior?
Thanks

Reply:
if deploy is not defined, then go to local and get the port_alias, see line 87. this works because sonic-mgmt containers installed all sonic-device-data package as well. It is the same as DUT. Ideally, we do not need to go to the DUT to get the port alias.

Reply:
Thanks, will try to solve it this way.